index.html

```diff
@@ -403,10 +403,10 @@ <h4 class="desktop-only">FACEBOOK</h4>
 
             </div>
 
-         </div>
+         </main>
```

Comment:
- I recommend that you don't forget to change your </main> tag to </section> to follow good HTML practices.